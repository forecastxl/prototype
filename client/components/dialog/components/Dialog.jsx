import { Card } from 'material-ui/Card'
import styled from 'styled-components'

const Dialog = styled(Card)`
  width: 25rem;
  margin: 5rem auto 0;
`

export default Dialog
