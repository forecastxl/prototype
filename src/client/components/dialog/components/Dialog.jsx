import { Card } from 'material-ui/Card'
import styled from 'styled-components'

const Dialog = styled(Card)`
  margin: 5rem auto 0;
  width: 25rem;
`

export default Dialog
