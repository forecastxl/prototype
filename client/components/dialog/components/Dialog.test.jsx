import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Dialog from './Dialog'

describe('<Dialog />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Dialog />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
