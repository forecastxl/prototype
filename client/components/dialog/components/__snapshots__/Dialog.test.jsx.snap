// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dialog /> renders correctly 1`] = `
<Card
  className="ebtpaV"
  expandable={false}
  expanded={null}
  initiallyExpanded={false}
/>
`;
